src/debugger/scriptImporter.ts

```diff
@@ -86,4 +87,14 @@ export class ScriptImporter {
                 return new FileSystem().writeFile(sourceMappingLocalPath, updatedContent);
             });
     }
+
+    /**
+     * Changes the port of the url to be the one configured as this.packagerPort
+     */
+    private overridePackagerPort(urlToOverride: string): string {
```

Comment:
This has nothing to do multiple VS Code instances. This is related to the fact that if we use port forwarding the RN App thinks that the packager is at localhost:8081 when it's actually on localhost:8090 in the Computer, so we need to change the port when we download the scripts to actually find them.